### ISSUE TITLE
2.1.2 wont run, error about no module Controllers

### ISSUE THREAD

**Issue by noideaman** [ZenithVal/OSCLeash#24]
Error output
```
C:\Users\noideaman\AppData\Local\Programs\OSCLeash>OSCLeash.exe
Traceback (most recent call last):
  File "C:\Python311\Lib\site-packages\cx_Freeze\initscripts\__startup__.py", line 124, in run
  File "C:\Python311\Lib\site-packages\cx_Freeze\initscripts\console.py", line 16, in run
  File "..\OSCLeash.py", line 7, in <module>
ModuleNotFoundError: No module named 'Controllers'
```
but running with python works just fine, i only installed python_osc over pip no extra modules

I have tried adding packages and modules to the build config, but to no avail so far.

**Comment by noideaman** [comment]
This same error pops up on my vr box which does not have python installed, with the same C:\python311 path

**Comment by noideaman** [comment]
Found the issue.
Will need to fix realtive paths, as the the script builds out of the "build" folder. so it cant find the "controller" folder as it's working dir is one level too high

**Comment by ZenithVal** [comment]
Fixed in v2.1.3
